src/index.ts

```diff
@@ -1,19 +1,19 @@
-import webpack from 'webpack';
+import extend from 'extend';
 import path from 'path';
 import { validate } from 'schema-utils';
+import { Schema } from 'schema-utils/declarations/validate.js';
 import svgstore from 'svgstore';
-import extend from 'extend';
+import type { Chunk, Compilation, Compiler, Module, NormalModule, sources } from 'webpack';
+import webpack from 'webpack';
 import templatePreview from './preview.js';
-import unTypedSchemaOptions from './schemas/plugin-options.json' assert { type: 'json' };
-import type { Compiler, Compilation, NormalModule, Chunk, Module, sources } from 'webpack';
-import { Schema } from 'schema-utils/declarations/validate.js';
+import unTypedSchemaOptions from './schemas/plugin-options.json' with { type: 'json' };
```

Comment:
It seems that ESLint does not yet support the syntax. The proposal is in stage 3 and ESLint is waiting stage 4

https://github.com/eslint/eslint/discussions/15305

Reply:
But it works with assert? Is that stage 4?